lib/fprotect/Kconfig

```diff
@@ -10,3 +10,12 @@ config FPROTECT
 	help
 	  Use hardware peripherals (BPROT, ACL, or SPU) to write-protect
 	  regions of flash until next reset.
+
+if FPROTECT
+config FPROTECT_BLOCK_SIZE
+	int "Block size of fprotect"
+	default 32768 if SOC_NRF9160 || SOC_NRF5340_CPUAPP
```

Comment:
53 has 16k SPU sections if I'm not mistaken.

Reply:
You are correct.